containers/EmpSponsors.ts

```diff
@@ -72,8 +75,12 @@ const useEmpSponsors = () => {
   // We set the poll interval to a very slow 5 seconds for now since the position states
   // are not expected to change much.
   // Source: https://www.apollographql.com/docs/react/data/queries/#polling
+  let clientName = "UMA";
+  if (network !== null) {
+    clientName += network.chainId.toString();
+  }
   const { loading, error, data } = useQuery(EMP_DATA, {
-    context: { clientName: "UMA" },
+    context: { clientName: clientName },
```

Comment:
nit: if the key and value is the same then do you need to include both? could this not simply be:
```
{ clinetName }
```

Reply:
Yeah but I like explicitly defining it so that the dev knows that `clientName` is the name of the key. I'll change `clientName` to `subgraphToQuery`